### ISSUE TITLE
Metrics only print for training, not validation

### ISSUE THREAD

**Issue by nikcheerla** [ncullen93/torchsample#39]
Hi,

Love this API and the fact that it's so similar to Keras's! One minor issue I've seen -- metrics are only evaluated for training, and not validation, so you can't see validation top-k accuracy and other metrics. Is this a feature that will be added soon?

**Comment by achaiah** [comment]
@nikcheerla   Are you certain?  Looking at the code it looks like validation metrics are being computed (around line 620) in `module_trainer.py`:
```
             if has_validation_data:
                    val_loss = self.evaluate_loader(val_loader,
                                                    cuda_device=cuda_device)
                    if self._has_metrics:
                        val_loss, val_metric_logs = val_loss
                        epoch_logs.update(val_metric_logs)
                    epoch_logs['val_loss'] = val_loss
                    self.history.batch_metrics['val_loss'] = val_loss
                
                # END OF EPOCH
```

**Comment by nikcheerla** [comment]
@achaiah Huh, it does look like its computing validation metrics. When I run the basic MNIST example though (at https://github.com/ncullen93/torchsample/blob/master/examples/mnist_example.py) with the exact same code, I don't see any validation metrics printed ...

**Comment by recastrodiaz** [comment]
@ncullen93 I see that you have recently forced merge to master. Is this still an issue? If so, do you want me to update the PR?
